lib/foreman_maintain/concerns/base_database.rb

```diff
@@ -52,6 +52,24 @@ def dump_db(file, config = configuration)
         execute!(dump_command(config) + " > #{file}", :hidden_patterns => [config['password']])
       end
 
+      def restore_dump(file, localdb, config = configuration)
+        if localdb
+          dump_cmd = "runuser - postgres -c 'pg_restore -C -d postgres #{file}'"
+          execute!(dump_cmd)
+        else
+          # TODO: figure out how to completely ignore errors. Currently this
+          # sometimes exits with 1 even though errors are ignored by pg_restore
+          dump_cmd = base_command(config, 'pg_restore') +
+                     ' --no-privileges --clean --disable-triggers -n public ' \
+                     "-d #{config['database']} #{file}"
+          execute!(dump_cmd, :valid_exit_statuses => [0, 1])
+        end
+      end
+
+      def restore_pg_globals(pg_globals, config = configuration)
+        puts base_command(config, 'psql') + " -f #{pg_globals} postgres 2>/dev/null"
```

Comment:
Did you meant to execute this? If so add also `:hidden_patterns => [config['password']]` to hide credentials from the logs